User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,80 @@
+# Survey questions
+
+## Readme
+
+This survey aims at evaluating users’ experiences with ESMValTool and
+discovering users’ needs.  It is meant for the users of ESMValTool in all levels
+of expertise. Some criteria to design the questions:
+
+- It does not take more than 10 (?) minutes
+- It does not have overlapping questions
+- It does not include questions with yes/no answers
+- It does not collect personal data, but it has some profiling questions
+- It covers different aspects of ESMValTool like application, usage, documentation and community
+
+The survey can be implemented in a survey app like surveymonkey that does not
+collect personal data. The owner is “user engagement team”. They will
+edit/run/evaluate the survey and communicate the results to other teams if
+needed.
+
+## Survey content
+
+Hi! We in [ESMValGroup](https://www.esmvaltool.org/), would love to hear about
+your research and experiences using ESMValTool. Here, there are a few questions
+which help to create your story. Your answers will help us to improve ESMValTool
+and hence enhance the science that you conduct. All responses are recorded
+anonymously. Let’s begin:
+
+**Describe your research/case study and challenges:**
+
+1. Can you describe your research/case study?
+2. Can you underline the challenges in your research/case study?
```

Comment:
Maybe we want to add here "... study with respect to data analysis and data handling" or something? So that it becomes clear that we want to know challenges that we could help with with ESMValTool.

Reply:
I agree with @hb326 - I wonder if we could say software and data handling/processing challenges or something?